themes/classic/_dev/js/cart.js

```diff
@@ -168,6 +168,39 @@ $(document).ready(() => {
     });
   };
 
+  //Hide 'have a code promo' text if displayed when 'close' is
+  let isPresent = $('.promo-code-button').length,
+      nameLength = 0;
+
+  function labelCodeDisplayed() {
+    $('.promo-code-button').not('.cancel-promo').hide();
+    
+    $('.cancel-promo').on('click',  (e) => {
```

Comment:
Shouldn't all classes used by js have a `js-` prefix? I see a lot of javascript executed on common class names. Wouldn't this break stuff for child themes? Keeping non `js-` prefixed classes unchanged is not mandatory and many developers may have removed or changed class names in their templates.

Reply:
I'm really not in favor to add a `js-` for class with js only, what happened if we improve theme and use a css on this class? More if a child theme use this `js-` for a css usage.
Keep thinking that a class is like an id, just a selector, not only for css.
It's really overkill and add characters for nothing :/